.circleci/config.yml

```diff
@@ -28,11 +28,21 @@ store-log-artifacts: &store-log-artifacts
     path: log
     destination: log
 
+save-flow-typed-cache: &save-flow-typed-cache
+  save_cache:
+    key: flow-typed-{{ checksum "package-lock.json" }}
```

Comment:
Is it depending only on `package-lock.json`? Not also our libraries? I think both but I might be mistaken

Reply:
It only depends on 3rd party libs. As long as we don't upgrade any library, typings won't have to change.
What do you mean exactly with _our libraries_ ?

Reply:
I think he means our code. Those are gonna be always in our codebase.